DB/db.sql

```diff
@@ -67,18 +57,16 @@ CREATE TABLE appartenenza (
        FOREIGN KEY (opera) REFERENCES opera(id) ON DELETE CASCADE
 );
 
-CREATE TABLE commento (
+CREATE TABLE recensione (
        timestamp TIMESTAMP,
        utente VARCHAR(30),
-       testo VARCHAR(200) NOT NULL,
+       commento VARCHAR(200),
```

Comment:
In questo caso il commento può essere NULL, ci vanno bene i commenti vuoti con solo le stelline? In caso positivo lascia così altrimenti bisogna mettere NOT NULL

Reply:
Avrebbe senso, visto il fatto che anche su google maps ci sono recensioni senza commento. Per esempio rovesti o come si chiama per synclab. Poi essendo opere non so se abbia senso, ma secondo me si

Reply:
Se per noi ha senso va bene lasciare così allora